metro.css

```diff
@@ -1,7 +1,184 @@
 @import url('https://fonts.googleapis.com/css2?family=Fugaz+One&family=Monoton&display=swap');
 *{
-    margin: 0;
```

Comment:
you have two different margins
Please choose one
More so, margin 15px is too much